# ancestral_retention_signals_cattle

Domestic cattle (Bos taurus and Bos indicus) underpin food security and livelihoods worldwide but face intensifying pressures from climate change, infectious disease, and inconsistent feed supplies. African and European indigenous cattle provide a natural comparative framework spanning gradients of climate, pathogen burden, and husbandry, and possess genomic mosaics comprising African taurine, European taurine, and indicine ancestry. We analyzed whole-genome sequences from 519 cattle across 24 African and European indigenous populations and 117 publicly available genomes from Africa, Asia, Europe, and the Americas. This dataset reveals admixture mosaics among major lineages and identifies 36 candidate genes exhibiting adaptive retention of ancestral alleles associated with response to heat stress (e.g., HSPA12B, DDIT3), immunity (IRAK3), productivity (ACSF3), and reproductivity (SSMEM1, SPEF1). Our study suggests that historical admixture introduced variation shaped by local ecological selection, clarifying how environmental heterogeneity drives the retention of advantageous alleles and informing sustainable breeding and diversity conservation.

# data availability
Population‐level variant calls (VCF files) for the 636 indigenous and reference cattle analyzed in this study have been deposited in the European Variation Archive (EVA) under accession number PRJEB102975 (ERP184337). Whole-genome sequences for the 519 indigenous European and African cattle analyzed in this study are deposited in the European Nucleotide Archive (ENA) under PRJEB90816 (European) and PRJEB90914 (African). Publicly available variants used as references were obtained from the 1000 Bull Genomes Project (Run 9; PRJNA391427, ERZ14211345) and African Genomic Reference Resource PRJEB74565 (Muturu). Sample identifiers for the ancestry reference panels (AFT, N = 34; EUT, N = 29; AFI, N = 25; AAI, N = 29) are listed in Supplementary Table 1. Processed data products, including sample- and variant-level summary statistics, admixture statistics, and dating summaries are provided as Supplementary Tables. Supplementary Data 1: Genome-wide haplotype-mosaic files for locus-level inspection, available in the same repository under Supplementary_Data/Mosaic_*. Data are provided for two AFT reference configurations: N’Dama + Muturu and Muturu only. Supplementary Data 2: Lists of putative ancestry-retention genes with breed retention percentages, available under Supplementary_Data/putative_genes.
If you have any questions about the process and article, please feel free to contact me (junxin.gao@wur.nl).

## Please cite us

If you use this pipeline or data, please cite:

**Gao J., et al.**  
*Distinct adaptation and ancestral retention signals in African and European indigenous cattle genomes.*  
*Communications Biology*, in press.
